src/templates/healthcert/memo/pdtV2Memo.tsx

```diff
@@ -39,9 +40,9 @@ const MemoResultSection: React.FC<{
       <p>To whom it may concern:</p>
       <p>
         The above-mentioned has undergone {memoInfo.observationTestTypeDisplay}{" "}
-        for COVID-19 using a {memoInfo.specimenSwabTypeDisplay} on{" "}
-        {memoInfo.specimenCollectionDateTime}, by {memoInfo.organizationLhpName}{" "}
-        and has tested{" "}
+        for COVID-19 using a {memoInfo.specimenSwabTypeDisplay}
+        on {memoInfo.specimenCollectionDateTime}, {memoInfo.modality ?? ""} by{" "}
```

Comment:
Think we should use `||` instead:

```suggestion
        on {memoInfo.specimenCollectionDateTime}, {memoInfo.modality || ""} by{" "}
```

Consideration: In case modality is ever an empty string `""`

```javascript
console.log("" ?? "value") // ""
console.log("" || "value") // "value"
```